apps/dashboard/src/@/components/blocks/FormFieldSetup.tsx

```diff
@@ -0,0 +1,36 @@
+import { Label } from "@/components/ui/label";
+import { ToolTipLabel } from "@/components/ui/tooltip";
+import { AsteriskIcon, InfoIcon } from "lucide-react";
+
+export function FormFieldSetup(props: {
+  id: string;
```

Comment:
probably copy the `htmlFor` name here to make it clear what it is used for?

Reply:
updated